clang/lib/3C/DeclRewriter.cpp

```diff
@@ -230,6 +242,8 @@ void DeclRewriter::rewriteFieldOrVarDecl(DRType *N, RSet &ToRewrite) {
   } else if (VisitedMultiDeclMembers.find(N) == VisitedMultiDeclMembers.end()) {
     std::vector<Decl *> SameLineDecls;
     getDeclsOnSameLine(N, SameLineDecls);
+    if (isInlineStruct(SameLineDecls))
+      SameLineDecls.erase(SameLineDecls.begin() + 0);
```

Comment:
Why the `+0`? Intuitively it should be a no-op, but I can't say I know the specifics of arithmetic on C++ iterators.

Reply:
You're right it's a no-op. Mistake to include it